src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodsValue, setGoodsValue] = useState('Jam');
+  const [title, setTitle] = useState(` is selected`);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {goodsValue}
+        {title}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+        {(title !== 'No goods selected')
+          && (
             <button
-              data-cy="AddButton"
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => {
+                setTitle('No goods selected');
+                setGoodsValue('');
+              }}
+            />
+          )
+        }
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      </h1>
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        {goods.map(good => (
+          <tbody>
+            <tr
```

Comment:
1. map goods inside `<tbody>`
2. create variable isSelected since you use this condition several times
3. add a React `key` for an item your render at a list

```suggestion
        <tbody>
          {goods.map(good => {
            const isSelected = goodsValue === good
            return (
              <tr
                key={good}
```